_woc/Weekly_blogs.md

```diff
@@ -0,0 +1,65 @@
+  actions:
+    - label: "View Repository on Github"
+      url: "https://github.com/khanfarhan10/TextSentimentAnalysis"
+author: Dinesh_Kumar
+use_math: true
+---
+## Introduction 
+Blah Blah Blah. 
+My name is **Dinesh Kumar**.I’m an inquisitive thinker and a Python Developer who has his commitments
+towards data science. I am dedicated and always eager to learn more and
+more about data science, machine learning and artificial intelligence
+related knowledge.
+### About You
+I'm **Dinesh Kumar** Currently pursuing my btech from **NIET** greater noida in **Computer Science** department.My Hobbies are to play Cricket and love to listen songs.
+#### Education
+- I have completed my High school Qualification from **Ram-eesh International School** in 2018.
+- And currently I am pursuing my Btech From **Noida Institute of Engineering and Technology** in greater noida from Computer Science Branch.
+- Currently i am in **3rd year** and passout by 2022.
+#### Projects
+- I have prepared many models like **Supervised models**, **Unsupervised model**, **Decision tree model**.
+- I have also done **EDA** on a mall details.
+- In october month i also completed my **Internship** in **Spark's Foundation** as **Data Science and Buisness Analytics**.
+
+#### Why WOC?
+After i completed my internship i think to search for more to learn extra in this field so i found about **WOC(winter of code)** and i read about this and feel that i should apply in this to learn about how to build various models and will get to interact to different people.So overall i am happy now after joining this and every day i learn new things.
+
+## Welcome to my Winter of Code Blog!
+- hello everyone!
+  My name is **Dinesh Kumar**. Currently i am pursuing my **Btech** from **NIET** in **Computer Science** department from Greater noida. This is my **first week** blog.
+  I have been Selected for **WOC 2020** at sub-org **DSC-IEM** and the project is to make model on **TEXT SENTIMENT ANALYSIS**
+  url:-https://github.com/khanfarhan10/TextSentimentAnalysis
+
+## what i implement in this week?
+Firstly i searched for dataset on which i will gona train my model. I found **Amazon food reviews** dataset then i import it and start doing **EDA** and done some **Text Cleaning**.I train it using **BOG** n-grams and **TF-IDF** and i used **NAIVE BAYES** and **LOGISTIC REGRESSION** Algorithms.
+
+## Upcoming Plans for Next Week?
+I will work more on to improve model performance and further firstly deploy it on local server using **streamlit** and then deploy it globally using **Heroku**.
+
+## Did you get stuck anywhere?
+I stuck when i was training my model. its performance not coming upto mark so i searched and found about **hyper parameter** tuning so i used optimal alpha to find good hyper parameter and then trained it so now it comes nice one its **accuracy score is 90.86%** and **F1 score is 0.81** and after that i also applied n-gram to enhance it performance.
```

Comment:
Rest is great! I appreciate the efforts!!!

Reply:
thanks

Reply:
Add some external links if you wish to and make he page more interesting!